### PR TITLE
Add prefix for getOpKeys

### PR DESCRIPTION
**Pull request by sydhds**

### CODE REVIEW

File: assembly/massa-sc-runtime/op_fn.ts [hunk 1]
```diff
@@ -26,4 +27,13 @@ export function main(_args: string): void {
             // generateEvent(msg);
         }
     }
-}
\ No newline at end of file
+
+    // with prefix
+    let _prefix: Array<u8> = [127];
+    let prefix = StaticArray.fromArray(_prefix);
+    let keys2: Array<StaticArray<u8>> = getOpKeys(prefix);
```
Comment (damip) [inline assembly/massa-sc-runtime/op_fn.ts:34]: isn't it getOpKeysPrefix(prefix) ?

File: assembly/massa/datastore/main.ts [hunk 2]
```diff
@@ -2,7 +2,7 @@ import { print, generateEvent, hasOpKey, getOpData, getOpKeys } from "@massalabs
 
 export function main(_args: string): void {
 
-    let keys: Array<StaticArray<u8>> = getOpKeys();
+    let keys: Array<StaticArray<u8>> = getOpKeys(new StaticArray<u8>(0));
```
Comment (damip) [inline assembly/massa/datastore/main.ts:5]: here as well

File: assembly/massa/deploy_sc/deployer.ts [hunk 3]
```diff
@@ -2,7 +2,7 @@ import { generateEvent, getOpKeys, getOpData, createSC, call, functionExists } f
 import { Args } from '@massalabs/as-types';
 
 export function main(): void {
-    const keys = getOpKeys();
+    const keys = getOpKeys(new StaticArray<u8>(0));
```
Comment (damip) [inline assembly/massa/deploy_sc/deployer.ts:5]: here as well

File: assembly/massa/get_call_coins/main.ts [hunk 4]
```diff
@@ -11,7 +11,7 @@ import { createSC, generateEvent, getOpData, getOpKeys } from "@massalabs/massa-
 
 export function main(_args: string): void {
     // Create every SC available in the ExecuteSC operation datastore
-    const keys = getOpKeys();
+    const keys = getOpKeys(new StaticArray<u8>(0));
```
Comment (damip) [inline assembly/massa/get_call_coins/main.ts:14]: here as well

File: assembly/massa/local_call/main.ts [hunk 5]
```diff
@@ -3,7 +3,7 @@ import { Args } from '@massalabs/as-types';
 
 export function main(_args: string): void {
     generateEvent("event generated before the sc");
-    const keys = getOpKeys();
+    const keys = getOpKeys(new StaticArray<u8>(0));
```
Comment (damip) [inline assembly/massa/local_call/main.ts:6]: here as well

File: assembly/massa/local_execution/main.ts [hunk 6]
```diff
@@ -3,7 +3,7 @@ import { Args } from '@massalabs/as-types';
 
 export function main(_args: string): void {
     generateEvent("event generated before the sc");
-    const keys = getOpKeys();
+    const keys = getOpKeys(new StaticArray<u8>(0));
```
Comment (damip) [inline assembly/massa/local_execution/main.ts:6]: here as well

File: assembly/massa/nested_call/main.ts [hunk 7]
```diff
@@ -9,9 +9,9 @@
 
 import { createSC, generateEvent, getOpData, getOpKeys, print } from "@massalabs/massa-as-sdk";
 
-export function main(_args: string): void {
+export function main(_args: StaticArray<u8>): void {
     // Create every SC available in the ExecuteSC operation datastore
-    const keys = getOpKeys();
+    const keys = getOpKeys(new StaticArray<u8>(0));
```
Comment (damip) [inline assembly/massa/nested_call/main.ts:14]: here as well

File: assembly/massa/send_message/send_message.ts [hunk 8]
```diff
@@ -8,7 +8,7 @@ import { sendMessage, print, createSC, getOpKeys, getOpData } from "@massalabs/m
 export function main(name: string): void {
     // Create every SC available in the ExecuteSC operation datastore
     // Send an asynchronous message to every created SC
-    const keys = getOpKeys();
+    const keys = getOpKeys(new StaticArray<u8>(0));
```
Comment (damip) [inline assembly/massa/send_message/send_message.ts:11]: here as well

File: assembly/massa/set_bytecode_fail/main.ts [hunk 9]
```diff
@@ -7,13 +7,13 @@
  * which is ready to be send on Massa network node!
  **/
 
-import { getOpData, getOpKeys, Storage } from "@massalabs/massa-as-sdk";
+import { getOpData, getOpKeys, Storage, setBytecode } from "@massalabs/massa-as-sdk";
 
 export function main(_args: string): void {
     // Store every SC available in the ExecuteSC operation datastore overriding the previous one
-    const keys = getOpKeys();
+    const keys = getOpKeys(new StaticArray<u8>(0));
```
Comment (damip) [inline assembly/massa/set_bytecode_fail/main.ts:14]: here as well

File: package.json [hunk 10]
```diff
@@ -62,7 +62,7 @@
     "@massalabs/as": "1.1.1-dev.20221215083749",
     "@massalabs/as-transformer": "^0.1.2",
     "@massalabs/as-types": "^0.0.3-dev",
-    "@massalabs/massa-as-sdk": "^2.0.2-dev",
+    "@massalabs/massa-as-sdk": "../umbrella/massa-as-sdk",
```
Comment (damip) [inline package.json:65]: TODO: update

File: assembly/massa-sc-runtime/op_fn.ts (hunk 1 above)
Comment (sydhds) [reply assembly/massa-sc-runtime/op_fn.ts]: Fixed

File: assembly/massa/datastore/main.ts (hunk 2 above)
Comment (sydhds) [reply assembly/massa/datastore/main.ts]: Fixed

File: assembly/massa/deploy_sc/deployer.ts (hunk 3 above)
Comment (sydhds) [reply assembly/massa/deploy_sc/deployer.ts]: Fixed

File: assembly/massa/get_call_coins/main.ts (hunk 4 above)
Comment (sydhds) [reply assembly/massa/get_call_coins/main.ts]: Fixed

File: assembly/massa/local_call/main.ts (hunk 5 above)
Comment (sydhds) [reply assembly/massa/local_call/main.ts]: Fixed

File: assembly/massa/local_execution/main.ts (hunk 6 above)
Comment (sydhds) [reply assembly/massa/local_execution/main.ts]: Fixed

File: assembly/massa/nested_call/main.ts (hunk 7 above)
Comment (sydhds) [reply assembly/massa/nested_call/main.ts]: Fixed

File: assembly/massa/send_message/send_message.ts (hunk 8 above)
Comment (sydhds) [reply assembly/massa/send_message/send_message.ts]: Fixed

File: assembly/massa/set_bytecode_fail/main.ts (hunk 9 above)
Comment (sydhds) [reply assembly/massa/set_bytecode_fail/main.ts]: Fixed

**Review by modship — APPROVED** [review]

**Pull request merged**